specifyweb/frontend/js_src/lib/components/QueryBuilder/Export.tsx

```diff
@@ -87,18 +127,27 @@ export function QueryExportButtons({
         <QueryButton
           disabled={fields.length === 0}
           showConfirmation={showConfirmation}
-          onClick={(): void =>
-            doQueryExport(
-              '/stored_query/exportcsv/',
-              userPreferences.get(
-                'queryBuilder',
-                'behavior',
-                'exportFileDelimiter'
-              )
-            )
-          }
+          onClick={(): void => {
+            selectedRows.size === 0
+              ? doQueryExport(
+                  '/stored_query/exportcsv/',
+                  userPreferences.get(
```

Comment:
since you now call `userPreferences.use()` you don't need to call `userPreferences.get` here anymore and can just use `separator` variable here